compose-network/mirror-node/application.yml

```diff
@@ -18,6 +18,8 @@ hedera:
         secretKey: minioadmin
         bucketName: hedera-streams
         endpointOverride: http://minio:9000
+        local:
+          deleteAfterProcessing: true
```

Comment:
Remove this, so we get the default from the mirror-node itself. If we need to enable it, will do so when starting the local-node itself and clean it after starting.

Reply:
This flag needs to be present in the context of the whole debug workflow. 
This is the only way, I think, to double check that the Mirror node is started with "deleting of the record files **disabled**"

If you want to clear this. We need to start a cleanup procedure on **localnode stop**